uberfire-extensions/uberfire-layout-editor/uberfire-layout-editor-client/src/main/java/org/uberfire/ext/layout/editor/client/components/columns/ColumnWithComponents.java

```diff
@@ -69,6 +70,9 @@
     private boolean selected = false;
     private boolean selectable = true;
 
+    @Inject
+    Event<LockRequiredEvent> lockRequiredEvent;
```

Comment:
Try not to use @Inject on class fields, use the constructor for that. Also make field private

Reply:
Done